src/main/scala/io/iohk/ethereum/extvm/VMServer.scala

```diff
@@ -151,7 +151,10 @@ class VMServer(messageHandler: MessageHandler) extends Logger {
       world = world,
       initialAddressesToDelete = Set(),
       evmConfig = vmConfig,
-      originalWorld = world
+      originalWorld = world,
+      // FIXME add access list to CallContext
```

Comment:
I created a ticket ETCM-1202 for this so we can add it there
```suggestion
      // TODO ETCM-1202 use access list from CallContext
```